robot_controll/Config.cpp

```diff
@@ -0,0 +1,13 @@
+#include "Config.h"
+
+YAML::Node Config::config = YAML::LoadFile("../config.yaml");
```

Comment:
@D0d0 It would be awesome if we could reload this on the go by calling something like `Config::reload` but that might be out of scope of this PR.